package.json

```diff
@@ -76,7 +76,7 @@
           "default": true,
           "description": "If only a single import name matches when importing the word on the cursor, automatically import it rather than showing the import picker."
         },
-        "vandelay.showNewVersionAlert": {
+        "vandelay.alertNewVersion": {
```

Comment:
Why did this change? If I actually need it to change, then I must migrate the setting name.